tests/plugins/textwatcher/manual/throttle.html

```diff
@@ -0,0 +1,37 @@
+<style>
+#logger {
+	height: 200px;
+	overflow-y: auto;
+}
+</style>
+<strong>Logger: </strong>
+<ul id="logger"></ul>
+<div id="editor1" ></div>
+
+<script>
+	var logger = document.getElementById( 'logger' ),
+		editor = CKEDITOR.replace( 'editor1', {
+		on: {
+			instanceReady: function() {
+				var counter = 0,
+					counterInterval = 100,
+
+					textWatcher = new CKEDITOR.plugins.textWatcher( editor, function( selectionRange ) {
```

Comment:
Formatting is pretty bad here. First, this empty line between `counterInterval` and `textWatcher` is a little misleading, but a bigger problem is missing indentation after opening the function, and wrong braces indentation (closing the function).